lib/activerecord-mysql-structure/active_record/structure_sql_sanitizer.rb

```diff
@@ -153,5 +184,36 @@ def sort_indices(lines)
 
       sorted_lines
     end
+
+    def sort_versions(lines)
+      versions = []
+      sorted_lines = []
+      in_versions = false
+      lines.each do |line|
+        if in_versions
+          if MIGRATION_VERSION_VALUE_REGEX.match(line)
+            # Collect the version values until we run out (generally due to the newlines at the end)
+            versions << line
+          else
+            # Once we have stopped finding version values, sort them and add them all in order,
+            # correcting the separator punctuation
+            versions.sort.each_with_index do |version, idx|
+              if (versions.size - 1) == idx
+                sorted_lines << version.gsub(MIGRATION_VERSION_SEPARATOR_REGEX, ';')
+              else
+                sorted_lines << version.gsub(MIGRATION_VERSION_SEPARATOR_REGEX, ',')
+              end
+            end
+            sorted_lines << line
+            in_versions = false
+          end
+        else
+          # Most of the file until the "insert into schema_migrations" line is unchanged
+          sorted_lines << line
+          in_versions = !!SCHEMA_INSERT_REGEX.match(line)
+        end
+      end
+      sorted_lines
```

Comment:
We should have a base case here where if we're still `in_versions = true` that we sort through and add the versions.